### PR TITLE
Add a selftest for the "lwaftr loadtest" command

### PR DESCRIPTION
**Pull request by teknico**
Add a selftest for the `snabb lwaftr loadtest` command. Part of #412, implements #754.

Also small changes to the `run` selftest.

### CODE REVIEW

File: src/program/lwaftr/tests/subcommands/loadtest_test.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+@unittest.skipUnless(SNABB_PCI0 and SNABB_PCI1, 'NICs not configured')
+class TestLoadtest(unittest.TestCase):
+
+    run_cmd_args = (
+        SNABB_CMD, 'lwaftr', 'run',
+        '--bench-file', '/dev/null',
+        '--conf', DATA_DIR / 'icmp_on_fail.conf',
+        # NUMA node #0.
+        '--cpu', '4',
```
Comment (wingo) [inline src/program/lwaftr/tests/subcommands/loadtest_test.py]: Given that SNABB_PCI0 might be on NUMA node 1, should we make this a parameter as well?  Or is it defined to be on NUMA node 0?  In any case probably we should parameterize the CPU; the server could have any number of cores on this socket.

File: src/program/lwaftr/tests/subcommands/loadtest_test.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        '--bench-file', '/dev/null',
+        # Something quick and easy.
+        '--program', 'ramp_up',
+        '--step', '0.1e8',
+        '--duration', '0.1',
+        '--bitrate', '0.2e8',
+        # NUMA node #1.
+        '--cpu', '10',
+        # Just one card for on-a-stick mode.
+        BENCHDATA_DIR / 'ipv4_and_ipv6_stick_imix.pcap', 'ALL', 'ALL', SNABB_PCI0,
```
Comment (wingo) [inline src/program/lwaftr/tests/subcommands/loadtest_test.py]: Similar concerns, and weird that PCI0 would have affinity for NUMA node 1!

Comment (teknico) [reply src/program/lwaftr/tests/subcommands/loadtest_test.py]: Weird indeed: it comes from the `snabb1` current config. Relevant excerpt from `snabb1`'s `/etc/nixos/configuration.nix`:
```
  services.snabb_bot.environment = ''
        export SNABB_PCI0=0000:82:00.0
        export SNABB_PCI1=0000:02:00.0
  '';
```
The `0000:82:00.0` NIC is connected to NUMA #1, and the `0000:82:00.0` NIC to NUMA #0. The env vars definition should probably be swapped.

File: src/program/lwaftr/tests/subcommands/loadtest_test.py (hunk 1 above)
Comment (teknico) [reply src/program/lwaftr/tests/subcommands/loadtest_test.py]: Possibly. We currently don't have env vars defined for the CPUs we run processes on.

**Comment by teknico** [comment]
Removed hard-coded CPU affinity settings as agreed upon.

**Review by wingo — APPROVED** [review]

**Pull request merged**